ui/src/app/creditcard/creditcard.component.css

```diff
@@ -0,0 +1,6 @@
+.creditcardLabels{
+  text-align: right;
+}
+.creditcardPaymentOptions{
+  text-align: center;
+}
```

Comment:
You could have used bootstrap alignment utilities if alignment is all you need. http://getbootstrap.com/docs/4.1/utilities/text/#text-alignment

Reply:
same with previous session. If there are set standards sure why not.